src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -0,0 +1,15 @@
+package traffic_simulation
+
+fun main(args: Array<String>) {
+
+    scenario(testList, testRoad)
+}
+
+fun scenario(vehicleList: List<Vehicle>, road: RoadNetwork){
+    if(road.checkForTrafficJam(road.calculateDemand(vehicleList))){
```

Comment:
Die Idee die Logik in `calculateDemand` und `checkForTrafficJam` aufzuteilen ist sehr gut! :+1: Außenstehende (wie hier `Main`) sollten sich damit aber nicht abgegeben im Detail müssen. Wenn das `RoadNetwork` noch eine Methode `simulateTraffic` (oder so) bekäme, könnte diese die beiden Methoden aufrufen.